### PR TITLE
Task1 completed

### PR DESCRIPTION
**Pull request by KrystianPlich**

### CODE REVIEW

File: excercsises/1_introduction/minimal.spec.js [hunk 1]
```diff
@@ -15,26 +15,41 @@ describe('Protractor workshop app', function() {
...
-
 	/**
 	 * Check http://angular.github.io/protractor/#/api?view=ElementFinder
 	 * to see how to select element for verification
 	 */
 
-	xit('should have "Example headline 1" carousel item after entering site', function(){	
-
+	it('should have "Example headline 1" carousel item after entering site', function(){	
+		var firstActiveItemTagText = element(by.className("active item")).element(by.className("carousel-caption")).element(by.tagName('h1'));
```
Comment (rafalszmit) [inline excercsises/1_introduction/minimal.spec.js]: try to write selectors as short as possible. You can also chain them like this:
`element(by.css(' .active item .carousel-caption h1'));`
assuming you are using css selector, if you want to find element by class you can just start with `.` if its id you can just use `#`

File: excercsises/1_introduction/minimal.spec.js [hunk 2]
```diff
@@ -43,15 +58,33 @@ describe('Protractor workshop app', function() {
...
+		expect(features).toEqual(expectedAtr);
 	});
 
 	/**
 	 * Check http://angular.github.io/protractor/#/api?view=ElementArrayFinder
 	 * to see how fileter function can be used select elements based on condition
 	 */	
 	it('should route to "Blog" pages after selecting link',function(){
-		var menuItems = element.all(by.css('ul.nav > li > a'));
+		var menuItems = element.all(by.css('ul.nav > li > a')).filter(function(elem, index) {
```
Comment (rafalszmit) [inline excercsises/1_introduction/minimal.spec.js]: at the end here you have `.first().click()` if you are already clicking, there is no need to create menuItems var, you are not using it. If the selector is correct and it returns menu items, there should not be more than one 'Blog' element, so using `.first()` may not be necessary.

File: excercsises/1_introduction/minimal.spec.js [hunk 3]
```diff
@@ -43,15 +58,33 @@ describe('Protractor workshop app', function() {
...
+			return elem.getText().then(function(text) {
+				return text === 'Blog';
+			});
+		}).first().click();
+		var newPageHeader = element(by.className('page-header')).element(by.tagName('h1'));
```
Comment (rafalszmit) [inline excercsises/1_introduction/minimal.spec.js]: not that it is wrong but you can also do it like this to e.g. check the current URL.
```
let pageTitle = browser.driver.getTitle();
expect(pageTitle).toBe('Protractor workshop | Blog');
```

File: excercsises/1_introduction/minimal.spec.js [hunk 4]
```diff
@@ -15,26 +15,41 @@ describe('Protractor workshop app', function() {
...
-		var menuItems = element.all(by.css('ul.nav > li > a'));		
+		var expectedItemsText = ["Home", "About", "Services", "Blog", "Contact"]
+		var menuItems = element.all(by.css('ul.nav > li > a'))//.then(function(items) {
+		// 	expect(items.length).toBe(5);
+		// 	items.forEach(function (item, i) {
+		// 		expect(item.getText()).toContain(expectedItemsText[i])
+		// 		expect(item.getAttribute('href'))
+		// 	})
+		// });
+		expect(menuItems.get(0).getText()).toBe(expectedItemsText[0]);
```
Comment (rafalszmit) [inline excercsises/1_introduction/minimal.spec.js]: maybe some loop would work here?

**Comment by KrystianPlich** [comment]
Code updated accordingly to all comments

**Review by rafalszmit — APPROVED** [review]

**Pull request merged**